src/main/java/org/mockito/internal/session/DefaultMockitoSessionBuilder.java

```diff
@@ -0,0 +1,33 @@
+package org.mockito.internal.session;
+
+import org.mockito.MockitoSession;
+import org.mockito.internal.framework.DefaultMockitoSession;
+import org.mockito.internal.util.ConsoleMockitoLogger;
+import org.mockito.quality.Strictness;
+import org.mockito.session.MockitoSessionBuilder;
+
+public class DefaultMockitoSessionBuilder implements MockitoSessionBuilder {
+
+    private Object testClassInstance;
+    private Strictness strictness;
+
+    @Override
+    public MockitoSessionBuilder initMocks(Object testClassInstance) {
```

Comment:
Not sure about "initAnnotated(this)". Looks kind of awkward, half-finished and not speaking to me on what is actually going on. Verb + noun would be my preference. Other options: "initFields" or "initAnnotatedFields". The more I think about it, the more I like "initMocks" :) I acknowledge that it is not 100% accurate but it is slick and simple. At the end of the day, we're working with mocks!

There is a separate question about ROI. Does changing this method name actually make any tangible difference in the quality of the API? There is some churn caused by deprecation of 9-year old method in a very mature library that has lots of users (all code, sample and documentation all over the world).

@bric3, I'd like to move on with your original idea, e.g. session().initMocks(this). If we want to change initMocks in both, session and MockitoAnnotations, let's move it to a separate ticket.